### ISSUE TITLE
[Feature Request]: Loop a specific portion of a video, similar to VLC’s functionality.

### ISSUE THREAD

**Issue by BenLH77** [yuliskov/SmartTube#3843]
### Describe the feature you'd like to request

It would be useful if you could mark a specific start point (A) and end point (B) within a video like you see in VLC. This can be done through simple controls that let users set these points on a timeline or via input fields. 

Either way, thanks for building such a great app. 

### Describe the solution you'd like

![VLC Loop Section](https://github.com/user-attachments/assets/10050416-3d45-4fff-83fe-859d4c7347a2)

### Describe alternatives you've considered

Implementing this feature would greatly benefit users who need to review or focus on particular parts of a video. I believe it would add valuable functionality and improve the overall user experience.

**Comment by jkcapoeira** [comment]
Yes, this would be a great feature. I am learning to play guitar, and would like to have this feature, so I can repeat a to b until I learned that part.
